utils/src/main/java/uk/gov/di/authentication/utils/lambda/MFAMethodAnalysisHandler.java

```diff
@@ -176,11 +179,20 @@ private void fetchUserCredentialsAndProfileStatistics(
                 String email = userCredentialsItem.get(UserCredentials.ATTRIBUTE_EMAIL).s();
 
                 Map<String, AttributeValue> firstMfaMethod = null;
+                List<String> mfaMethodPriorityIdentifiers = new ArrayList<>();
                 if (userCredentialsItem.containsKey("MfaMethods")) {
                     List<AttributeValue> mfaList = userCredentialsItem.get("MfaMethods").l();
                     if (!mfaList.isEmpty()) {
                         firstMfaMethod = mfaList.get(0).m();
                     }
+                    for (AttributeValue mfaMethodValue : mfaList) {
+                        Map<String, AttributeValue> mfaMethod = mfaMethodValue.m();
+                        String mfaMethodPriority =
+                                Optional.ofNullable(mfaMethod.get("PriorityIdentifier"))
+                                        .map(AttributeValue::s)
+                                        .orElse(null);
```

Comment:
This will merge two states
1. the PriorityIdentifier value being null
2. the key PriorityIdentifier not existing